Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 def config = [
-  devDeploymentTimeout: "300s",
-  prodDeploymentTimeout: "300s",
+  devDeploymentTimeout: "600s",
+  prodDeploymentTimeout: "600s",
```

Comment:
Doubled these since I'm not sure what a true safe value is, but we don't want timeouts...